src/util/build/Build.cpp

```diff
@@ -23,19 +23,22 @@
 
 namespace util::build {
 
-static constexpr char versionString[] = "@CLIO_VERSION@"; // NOLINT(readability-identifier-naming)
+#ifndef CLIO_VERSION
+#error "CLIO_VERSION must be defined"
+#endif
+static constexpr char versionString[] = CLIO_VERSION;
 
 std::string const&
 getClioVersionString()
 {
-    static std::string const value = versionString; // NOLINT(readability-identifier-naming)
+    static std::string const value = versionString;  // NOLINT(readability-identifier-naming)
```

Comment:
For example, clang-format started treating this as all other source files and there should be 2 spaces before comment